DigitalLearningSolutions.Data.Tests/TestHelpers/UserTestHelper.cs

```diff
@@ -120,9 +120,9 @@ public static class UserTestHelper
         public static AdminUser GetDefaultCategoryNameAllAdminUser()
         {
             return GetDefaultAdminUser(
-                id: 11,
-                centreId: 59,
-                centreName: "Guy's and St Thomas' NHS Foundation Trust",
+                11,
+                59,
+                "Guy's and St Thomas' NHS Foundation Trust",

```

Comment:
These are not intentionally removed. Resharper finds the labels on these three redundant because they are in exactly the same order as the constructor's parameters. So it automatically removes them when cleaning up the code. When looking at this in VS, the parameter labels are inferred and displayed automatically (see picture), but I'm happy to add them back, maybe slightly reordering these so they don't vanish unexpectedly. 

![image](https://user-images.githubusercontent.com/80777689/134320885-56ec7005-aa40-4dc6-ac99-ad32c3985cd3.png)